Foundation/NSValue.swift

```diff
@@ -98,7 +98,7 @@ open class NSValue : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
     
     public convenience required init(bytes value: UnsafeRawPointer, objCType type: UnsafePointer<Int8>) {
-        if type(of: self) == NSValue.self {
+        if Swift.type(of: self) == NSValue.self {
```

Comment:
See https://lists.swift.org/pipermail/swift-users/Week-of-Mon-20170515/005442.html

Its due to the clash of the 'type' argument and the 'type' function call